apps/src/code-studio/pd/foorm/editor/components/FoormEntityLoadButtons.jsx

```diff
@@ -26,14 +27,33 @@ class FoormEntityLoadButtons extends React.Component {
     setLastSavedQuestions: PropTypes.func
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      latestVersionsOnly: false
+    };
+  }
+
   getDropdownOptions() {
-    return this.props.foormEntities.map((entity, i) => {
-      return this.renderMenuItem(
-        () => this.props.onSelect(entity['metadata']),
-        entity['text'],
-        i
-      );
-    });
+    return (
+      this.props.foormEntities
+        // sort entities alphabetically by name and version
+        .sort((a, b) => a['text'].localeCompare(b['text']))
+        // optionally filter out entities so that only the last unique version of a name remains
+        .filter((entity, i, array) =>
```

Comment:
I made it an optional thing and removed it from the library page, since I didn't add 'save as new version' there.. should I? Not sure how versioning works with libraries... it looks like the existing libraries were versioned by incrementing a number in the name.

Reply:
Yeah I think not showing it in library editor makes sense. Versioning is technically the same for Forms and Libraries (there are `version` columns in the Form and Library models, which are both derived from / stored to the JSON file paths representing their respective configurations), but I don't think what it means to version a library has been thought through in the same way that versioning a Form has.